File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { BluefinPayconexSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await BluefinPayconexSDK.test()
    equal(null !== testsdk, true)
  })

})

File: .sdk/tm/csharp/test/ExistsTest.cs
// BluefinPayconex SDK exists test.

using Xunit;

using BluefinPayconexSdk;

namespace BluefinPayconexSdk.Test;

public class ExistsTest
{
    [Fact]
    public void TestMode()
    {
        var testsdk = BluefinPayconexSDK.TestSDK(null, null);
        Assert.NotNull(testsdk);
    }
}
